examples/game/lib/game_demo_node.dart

```diff
@@ -467,6 +493,23 @@ class GameObjectFactory {
   Map<String,SoundEffect> sounds;
   Level level;
 
+  void addAsteroids(int numAsteroids, double yPos, double distribution) {
+    for (int i = 0; i < numAsteroids; i++) {
```

Comment:
It doesn't help you here, but List.generate() can sometimes be used for this type of thing.
https://api.dartlang.org/1.11.3/dart-core/List/List.generate.html